### ISSUE TITLE
code_on_top_scope warns about import

### ISSUE THREAD

**Issue by paran1** [rodjek/puppet-lint#581]
```
$ cat import.pp 
import 'foo'
$ ~/git/puppet-lint/bin/puppet-lint import.pp 
WARNING: code outside of class or define block - import on line 1
WARNING: code outside of class or define block - foo on line 1
```
Expected result: no warning from this check.

Tested with puppet-lint 3.0.0 and latest master (041b4303108309159e181092c883d2e6b8413649) with same result.

This issue is not 100% obviously wrong. Import was deprecated in puppet 3 and discontinued in puppet 4. If puppet-lint assumes that all manifests are for puppet 4, then I guess `import` could be considered 'code outside of class'.

However, I think this check should ignore `import`. Puppets own parser still accepts import in puppet 4, also not everybody uses puppet 4 yet.

```
$ puppet --version
4.5.2
$ puppet parser validate import.pp && echo OK
OK
$ puppet apply --noop --verbose import.pp 
Error: Evaluation Error: Use of 'import' has been discontinued in favor of a manifest directory. See http://links.puppetlabs.com/puppet-import-deprecation at /home/paran/puppet-lint-problem/import.pp:1:1 on node foo.example.com
$ 
```
If puppet-lint should warn about import I think a separate check for import, or perhaps also other deprecated things. Or ignore completely as this might not be considered a style issue?

**Comment by paran1** [comment]
I just realized that 3.0.0 is a branch rather than a tagged version, so please ignore that. The code_on_top_scope check is not in any released version.

**Comment by rnelson0** [comment]
This is by design; ALL code on the top scope is flagged when this check is left enabled.
